.gitignore

```diff
@@ -4,3 +4,5 @@ coverage
 tests/runtime
 coverage.xml
 .vagrant
+.idea
+.DS_STORE
```

Comment:
os and env-specific files. add this lines to your global `.gitignore` and be happier next year